src/xrGameSpy/xrGameSpy.cpp

```diff
@@ -1,16 +1,7 @@
 #include "stdafx.h"
 #include "xrGameSpy.h"
 
-void FillSecretKey(char* secretKey)
-{
-    secretKey[0] = 'L';
-    secretKey[1] = 'T';
-    secretKey[2] = 'U';
-    secretKey[3] = '2';
-    secretKey[4] = 'z';
-    secretKey[5] = '2';
-    secretKey[6] = '\0';
-}
+void FillSecretKey(char* secretKey) { strcpy(secretKey, GAMESPY_GAMEKEY); }
```

Comment:
```suggestion
void FillSecretKey(char* secretKey) { xr_strcpy(secretKey, GAMESPY_GAMEKEY); }
```

Reply:
Yes, you are right. It's sad there is no simple way to mark std-implementations as deprecated.

Reply:
We can: 816fca973cae3cf0a7df28b45b965f7fd92f6742